test/modules/core/lib/attribute-manager.spec.js

```diff
@@ -22,6 +22,9 @@
 import AttributeManager from '@deck.gl/core/lib/attribute-manager';
 import {GL} from 'luma.gl';
 import test from 'tape-catch';
+import global from 'global';
+
+const gl = global.glContext;
```

Comment:
Nits:
* A lot of the apps use `import {gl} from '@deck.gl/test-utils';`
* Have been trying to avoid the `global` npm module.